components/Chat/ChatSettings/ChatSettingsPanel/ChatTtsSettingsPanel.vue

```diff
@@ -0,0 +1,256 @@
+<template>
+  <v-card
+    flat
+    tile
+    class="mb-3 mx-3"
+  >
+    <!-- TTS Settings -->
+    <div>
+      <!-- No TTS Voice Warning -->
+      <v-alert
+        v-if="!ttsVoicesExist"
+        class="my-3"
+        type="warning"
+        text
+      >
+        No TTS voices found!
+      </v-alert>
+
+      <!-- TTS Voice Options -->
+      <div
+        v-if="ttsVoicesExist"
+      >
+        <!-- Enable TTS -->
+        <v-switch
+          v-model="useTts"
+          :disabled="!ttsVoicesExist"
```

Comment:
Seems redundant with the `v-if` on the same condition above.

Reply:
true